src/Apps/ArtAdvisor/05-Near-Object-Rail/App.tsx

```diff
@@ -0,0 +1,144 @@
+import { FC, useEffect, useState } from "react"
+import {
+  Spinner,
+  Spacer,
+  Box,
+  Select,
+  Text,
+  useToasts,
+  GridColumns,
+  Column,
+} from "@artsy/palette"
+import { Rail } from "Components/Rail/Rail"
+import { Artwork } from "Apps/ArtAdvisor/05-Near-Object-Rail/components/Artwork"
+import _ from "lodash"
+
+export type ArtworkType = {
+  _additional: {
+    id: string
+  }
+  title: string
+  imageUrl: string | null
+}
+
+type UserType = {
+  _additional: {
+    id: string
+  }
+  name: string
+}
+
+type UsersListType = {
+  value: string
+  text: string
+}
+
+export const App: FC = () => {
+  const [artworks, setArtworks] = useState<ArtworkType[]>([])
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const [usersList, setUsersList] = useState<UsersListType[]>([])
+  const [userId, setUserId] = useState<string>("")
```

Comment:
In this PR, the `userId` and `artworkId` are ALWAYS referring to their respective `weaviate` ids. I considered using `weaviateUserId` and `weaviateArtworkId`, but the longer name did not seem worth the tradeoff. I'm happy to clarify this more if we feel it would be an improvement.